### DISCUSSION TITLE
BadCertificate error when trying to use a template

### DISCUSSION THREAD

**Discussion by hspitia** [quarto-dev/quarto-cli#3956] [Q&A]
Hello all,

I'm trying to use a Quarto journal template (Elsevier), but I get an error related to certificates. The command I'm using is as follows:

```
quarto use template quarto-journals/elsevier
```

From which I get:

```
Sending fatal alert BadCertificate
ERROR: TypeError: error sending request for url (https://github.com/quarto-journals/elsevier/archive/refs/heads/main.tar.gz): error trying to connect: invalid peer certificate contents: invalid peer certificate: UnknownIssuer
```

I'm using a Mac with a corporate security setup (zero trust network access), which, I think, is causing this problem. Any idea about how to dig into the problem and solve it? I'll appreciate any help.

Thanks,

Hector

**Comment by mcanouil** [comment]
One workaround would be for you to clone and download manually.

Here, I think the issue comes from Deno which doesn't use your corporate certificate or possibly some issues with your corporate proxy.
Could you try to set `export DENO_TLS_CA_STORE=system` (https://deno.land/manual@v1.25.2/getting_started/setup_your_environment#environment-variables) in your bash/zsh terminal?

**Reply by hspitia** [reply]
Thanks a lot!
I use zsh. Unfortunately, it didn't work setting that variable, so I'll try cloning it manually.
Thanks again!

**Comment by SimonCoulombe** [comment]
worked for me, thanks!
